beacon-chain/core/validators/validator_test.go

```diff
@@ -132,7 +132,7 @@ func TestSlashValidator_OK(t *testing.T) {
 	state, err := v1.InitializeFromProto(base)
 	require.NoError(t, err)
 
-	slashedIdx := types.ValidatorIndex(2)
+	slashedIdx := types.ValidatorIndex(3)
```

Comment:
This had to change because now running all tests in the package assigns `2` to the proposer index, colliding with the slashed index, and in turn giving the wrong result.